src/theory/engine_output_channel.cpp

```diff
@@ -82,7 +82,7 @@ theory::LemmaStatus EngineOutputChannel::lemma(TNode lemma,
   d_engine->d_outputChannelUsed = true;
 
   PROOF({
-    bool preprocess = p & LemmaProperty::PREPROCESS;
+    bool preprocess = isPreprocess(p);
```

Comment:
Shouldn't it be `isLemmaPropertyPreprocess()` here?

Reply:
Yes, I'm using CI to check compiler errors for me unfortunately :)